server/chat/backend/agent/utils/persistence/context_manager.py

```
@@ -154,6 +161,49 @@ def _apply_summarization(self, messages, tool_capture):
                 processed.append(msg)
         return processed
 
+    def _redact_tool_messages(self, messages, *, user_id: str, session_id: str):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_redact_tool_messages`

(ANN202)